Task 3/Circular Linked list/Bucket Sort/bucket_sort.py

```diff
@@ -0,0 +1,41 @@
+# Python3 program to sort an array  
```

Comment:
Change this folder path to Bucket Sort instead of Linked List

Reply:
Yeah I have done that..Thanks for letting me know